### PR TITLE
fix clusterrolebinding

### PR DESCRIPTION
**Pull request by 2327**
small fix for deploy to another namespace than default

### CODE REVIEW

**Comment by joaquimrocha** [comment]
Thanks @2327 . The patch seems fairly simple but I'll ask @yolossn to review.
Can you (@2327 ) in the meanwhile follow our guideline for commit messages? Something like: `charts: Support deploying the ServiceAccount in different namespaces`

For more info see https://github.com/kinvolk/contribution/blob/master/topics/git.md#commit-format .

File: charts/headlamp/templates/clusterrolebinding.yaml [hunk 1]
```diff
@@ -16,5 +16,5 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: {{ include "headlamp.serviceAccountName" . }}
-  namespace: kube-system
+  namespace: {{ .Release.Namespace | default "kube-system" }}
```
Comment (yolossn) [inline charts/headlamp/templates/clusterrolebinding.yaml:19]: Good catch, I think the default value is not necessary just `.Release.Namespace` should do

```suggestion
  namespace: {{ .Release.Namespace }}
```

**Comment by ashu8912** [comment]
@yolossn, @joaquimrocha any updates on this??

**Comment by joaquimrocha** [comment]
Closing in favor of #509 , since the author is unresponsive.

**Pull request closed without merging**